spec/security-warnings-spec.js

```diff
@@ -75,6 +80,7 @@ describe('security warnings', () => {
           show: false,
           webPreferences: {
             webSecurity: false,
+            enableRemoteModule: false,
```

Comment:
again why? Can we either revert these changes to these unrelated tests or explain why it's now required to disable remote here.

Reply:
it's not required to disable remote here, but I wanted to test that the warnings still work without remote (that was the previous refactoring)

Reply:
Generated specs for remote disables as discussed offline